.github/workflows/scripts/authorValidation.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# set dependabot identifiers
+dependabotName="dependabot[bot]"
+committerName="web-flow"
+
+if [ $USER_NAME = "$dependabotName" ];
+then
+    echo "$dependabotName was detected as PR creator"
+
+    # get commits (for this PR) from github
+    wget --quiet --method GET -O commits.json $LINK
+
+    # extract from json response
+    committers=$( jq -r  '[.[] | .committer | .login] | unique' commits.json )
+    authors=$( jq -r  '[.[] | .author | .login] | unique' commits.json )
+    committersLength=$( jq '. | length' <<< "$committers" )
+    authorsLength=$( jq '. | length' <<< "$authors" )
+
+    echo "Committer count: $committersLength"
+    echo "Author count: $authorsLength"
+    # check if there were more than one author (not only dependabot committed)
+    if [ $committersLength -gt 1 ] || [ $authorsLength -gt 1 ];
+    then
+        echo 'There are too many committers and/or authors for a valid PR branch by $dependabotName'
+        exit 1
+    else
+        # extract the only left committer & author
+        committer=$( jq -r '.[0]' <<< $committers )
+        author=$( jq -r '.[0]' <<< $authors )
+        echo "Committer: $committer"
+        echo "Author: $author"
+
+        # check again if committer is same as dependabot name to be sure
+        if [ $committer != "$committerName" ] || [ $author != "$dependabotName" ];
```

Comment:
Propsal: `check if committer and author values have expected dependabot identifier values`